apis/slo/v1alpha1/nodeslo_types.go

```diff
@@ -177,8 +177,10 @@ type ResourceThresholdStrategy struct {
 
 	// if be CPU RealLimit/allocatedLimit > CPUEvictBESatisfactionUpperPercent, then stop evict BE pods
 	CPUEvictBESatisfactionUpperPercent *int64 `json:"cpuEvictBESatisfactionUpperPercent,omitempty"`
-	// if be CPU (RealLimit/allocatedLimit < CPUEvictBESatisfactionLowerPercent and usage nearly 100%) continue CPUEvictTimeWindowSeconds,then start evict
+	// if be CPU (RealLimit/allocatedLimit < CPUEvictBESatisfactionLowerPercent and usage >= CPUEvictBEUsageThresholdPercent / 100) continue CPUEvictTimeWindowSeconds, then start evict
```

Comment:
Sorry for not clarifying the problem clearly. We expect to correct the comments related to XXXPercent.

e.g.

```
if be CPU RealLimit/allocatedLimit > CPUEvictBESatisfactionUpperPercent/100
if be CPU (RealLimit/allocatedLimit < CPUEvictBESatisfactionLowerPercent/100 and usage ...
```

Reply:
what about an individual PR to do that, its not topic related

Reply:
updated